syntax = "proto3";

package brand_cls;

service ImageProcessingService {
  rpc ProcessImage (ImageRequest) returns (ImageResponse);
}

message ImageRequest {
  string token = 1;
  bytes image_data = 2;
}

message ImageResponse {
  string car_brand = 1;
  float car_brand_score = 2;
}
